packages/manager/apps/web/client/app/hosting/database/order/public/hosting-database-order-public.controller.js

```diff
@@ -39,6 +44,41 @@ export default class HostingDatabaseOrderPublicCtrl {
     return selectEngineVersion?.planCode || selectVersion?.planCode;
   }
 
+  getProductName() {
+    const { productName } = this.model.dbCategory;
+    return productName;
+  }
+
+  getServiceNameToAddProduct() {
```

Comment:
```suggestion
  get serviceNameToAddProduct() {
```

Reply:
I fact the function isn't use in this ctrl but need the model for to change the param to product offer module that is one bind. So i need to pass the function.bind(ctrl)